### PR TITLE
Add BuildScheduler selector.

### PR DESCRIPTION
**Pull request by matejonnet**
@slaskawi Do you find ok the renaming build-driver config to more general?

### CODE REVIEW

File: build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildScheduler.java [hunk 1]
```diff
@@ -58,13 +57,18 @@
 
     private BpmCompleteListener bpmCompleteListener;
 
-    private final URL instanceUrl;
```
Comment (slaskawi) [inline build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildScheduler.java]: Why URL was bad? Currently we miss the validation piece...

File: common/src/main/resources/pnc-config.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
 "@class":"ModuleConfigJson","name":"pnc-config",
         "configs":[
           {
+            "@module-config":"system-config",
+            "buildDriverId":"termd-build-driver",
```
Comment (slaskawi) [inline common/src/main/resources/pnc-config.json]: You probably need to change also NCL scripts.

File: build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildScheduler.java (hunk 1 above)
Comment (matejonnet) [inline build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/bpm/BpmBuildScheduler.java]: It required valid URL also if you don't use BPM and it caused an error due to our env variable reference. Validation is still done in later stage, when you are connecting to BPM server. I prefer the first option but a compromise is required.

File: common/src/main/resources/pnc-config.json (hunk 2 above)
Comment (matejonnet) [inline common/src/main/resources/pnc-config.json]: Changing autodeploy one ...

**Comment by matejonnet** [comment]
Retest this please.

**Pull request merged**